### ISSUE TITLE
Wrong units in dsphwidgets.py

### ISSUE THREAD

**Issue by fanqo** [ndrs92/DesignSPHysics#10]
Hello, thanks for the great utility. After trying it, we found that in the file "dsphwidgets.py", some units seem to be wrong. They are 'self.ampl_label = QtGui.QLabel("Ampl (m)")' and  'self.phase_label = QtGui.QLabel("Phase (rad)")' and may be needed to change to 'self.phase_label = QtGui.QLabel("Phase (degrees)")' and 'self.ampl_label = QtGui.QLabel("Ampl (degrees)")'.  Please check if I was wrong.

**Comment by ndrs92** [comment]
Hello,

First of all thanks for the issue. The lines of code that you stated belong to a Rotational Sinusoidal Movement.

I've checked with the developers of DualSPHysics and indeed the Amplitude is specified in meters in the XML, so in the GUI of DesignSPHysics is correct.

For the Phase, it can be specified as degrees or radians in the XML. We decided that in the GUI the units that we want people to use are radians, being the more popular one. I'll add to the code the 'anglesunits' keyword to the XML exporter accordingly to ensure DualSPHysics uses the value as radians.

So, in summary, the units are already correct, and I'll change a bit of the code to ensure that.
